embeddedconfig/global.yaml.tmpl

```diff
@@ -422,3 +422,4 @@ otel:
     autoupdate_download: 1
     autoupdate_install: 1
     check_update: 1
+    broflake_fronted_roundtrip: 800000
```

Comment:
Linking to the discussion about this for posterity:
https://github.com/getlantern/engineering/issues/1723#issuecomment-2451975960